src/resources/postcss/components/_top-bar.pcss

```diff
@@ -43,12 +43,12 @@
 		display: flex;
 		flex: 1 1 60%;
 		align-items: center;
+	}
 
-		.tribe-common-h3 {
-			font-size: var(--font-size-6);
-			@media (--viewport-medium) {
-				font-size: var(--font-size-8);
-			}
+	.tribe-events__top-bar-today--title {
+		font-size: var(--font-size-6);
+		@media (--viewport-medium) {
```

Comment:
Are y'all doing any pcss linting?

Reply:
@paulmskim

Reply:
Not sure, I wasn't too worried about it tbh as we could go back later and lint the heck out of the pcss haha.